java/log_correlation/log4j2/build.gradle

```diff
@@ -10,11 +10,9 @@ repositories {
 }
 
 group = "io.opencensus"
-version = "0.16.0-SNAPSHOT"
+version = "0.17.0-SNAPSHOT"
```

Comment:
Is this the version of opencensus-experiments and that's why it doesn't need to match L15?

Reply:
Yes, that is the version of the demo.  I just used the same version as the opencensus-java repository, but I don't think it matters since this project won't be released.

Reply:
Neat, thx.